firefox-ios/Client/Frontend/Browser/BrowserViewController/Extensions/BrowserViewController+ReaderMode.swift

```diff
@@ -4,12 +4,23 @@
 
 import WebEngine
 import Shared
+import Common
 
 extension BrowserViewController: ReaderModeDelegate {
     func readerMode(_ readerMode: ReaderMode, didChangeReaderModeState state: ReaderModeState, forTab tab: Tab) {
         // Update reader mode state if is the selected tab. Otherwise it will update once is active
-        if !isToolbarRefactorEnabled, tabManager.selectedTab === tab, !isToolbarRefactorEnabled {
-            urlBar.updateReaderModeState(state)
+        if tabManager.selectedTab === tab {
```

Comment:
Nit: can we convert this into a `guard` to not have so many nested if statements?